lbrynet/lbry_file/client/EncryptedFileDownloader.py

```diff
@@ -28,21 +29,22 @@ def __init__(self, stream_hash, peer_finder, rate_limiter, blob_manager,
         self.stream_info_manager = stream_info_manager
         self.suggested_file_name = None
         self._calculated_total_bytes = None
+        self.sd_hash = None
```

Comment:
This was moved from ManagedEncryptedFileDownloader in lbrynet/file_manager/EncryptedFileDownloader.py which inherits from this class